js/index.js

```diff
@@ -40,3 +40,66 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+var hEdit = document.querySelectorAll('a');
+hEdit[0].textContent = siteContent['nav']['nav-item-1'];
+hEdit[1].textContent = siteContent['nav']['nav-item-2'];
+hEdit[2].textContent = siteContent['nav']['nav-item-3'];
+hEdit[3].textContent = siteContent['nav']['nav-item-4'];
+hEdit[4].textContent = siteContent['nav']['nav-item-5'];
+hEdit[5].textContent = siteContent['nav']['nav-item-6'];
+hEdit[0].style.color = "green";
+hEdit[1].style.color = "green";
+hEdit[2].style.color = "green";
+hEdit[3].style.color = "green";
+hEdit[4].style.color = "green";
+hEdit[5].style.color = "green";
+
+let ctaEdit = document.getElementById("cta-img");
+ctaEdit.setAttribute('src', siteContent['cta']['img-src']);
+
+let ctaH1 = document.getElementsByTagName("h1");
+ctaH1[0].textContent = siteContent['cta']['h1'];
+
+let ctaButton = document.getElementsByTagName("button");
+ctaButton[0].textContent = siteContent['cta']['button'];
+
+let mainTextH4 = document.querySelectorAll('h4');
+mainTextH4[0].textContent = siteContent['main-content']['features-h4'];
+
+let mainP = document.querySelectorAll('p');
+mainP[0].textContent = siteContent['main-content']['features-content'];
+
+mainTextH4[1].textContent = siteContent['main-content']['about-h4'];
+mainP[1].textContent = siteContent['main-content']['about-content'];
+
+mainTextH4[2].textContent = siteContent['main-content']['product-h4'];
+mainP[2].textContent = siteContent['main-content']['product-content'];
+
+mainTextH4[3].textContent = siteContent['main-content']['vision-h4'];
+mainP[4].textContent = siteContent['main-content']['vision-content'];
+
+
+mainTextH4[5].textContent = siteContent['contact']['contact-h4'];
+mainP[5].textContent = siteContent['contact']['address'];
+mainP[6].textContent = siteContent['contact']['phone'];
+mainP[7].textContent = siteContent['contact']['email']; 
+
+mainP[8].textContent = siteContent['footer']['copyright'];
+
+
+let ctaImage = document.getElementById("cta-img");
+ctaImage.setAttribute('src', siteContent["cta"]["img-src"]);
+
+let middleImage = document.getElementById("middle-img");
+middleImage.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+// need to figure out why line 100 is the way that it is.
+
+var hEdit = document.querySelector('a');
+// get element content as string
+// append to the element's content
+hEdit.innerHTML += '<a>RANDOM STUFF</a>';
```

Comment:
You need to be using the prepend and appendChild methods here

Reply:
Here you're also just setting an element's html to have more html.  Prepend and appendChild will give you styling as well.

Reply:
I refactored the code and pushed it to use prepend, but the appendChild method isn't working, and I don't understand why. I'll research it some more because I'm having the same problem with my Components-I assignment.